src/components/Inputs.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Input } from 'indigo-react';
+import * as bip39 from 'bip39';
+
+import { validateNotEmpty, validateMnemonic } from 'lib/validators';
+import { DEFAULT_HD_PATH } from 'lib/wallet';
+
+const kExampleMnemonic = bip39.generateMnemonic();
+
+// const RequiredInput = advancedInput({
+//   WrappedComponent: Input,
+//   validators: [validateNotEmpty],
+// });
```

Comment:
What are these commented blocks doing here?

Reply:
These are the future inputs I've yet to convert to the new Input component — just kept them around for documentation.